core/options.c

```diff
@@ -2438,7 +2724,7 @@ show_dynamo_option_descriptions()
         print_file(STDERR, "-%-20s %s\n", command_line_option, description);        \
     }
 #include "optionsx.h"
-#undef OPTION_COMMAND
+# undef OPTION_COMMAND
```

Comment:
One or the other's indentation is off: if there's a containing ifdef both should have a space after `#` (or multiple if inside nested #ifs) else no space.

Reply:
Fixed.